File: Swift/primes.swift
import Foundation

let start = Date().timeIntervalSince1970

var primes: [Int] = []

for i in 0...14999 {
    var factors = 0
    if (i > 1){
        for j in 1...i {
            if i % j == 0 {
                factors += 1
            }
        }
    }
    if factors == 2 {
        primes.append(i)
    }
}

for i in 0...primes.count-1 {
    print("\(primes[i]) ")
}



let time = NSDate().timeIntervalSince1970 - start

let time_seconds = time

print("Completed in: \(time_seconds)s")

File: Javascript/primes.js
function primes() {
	let start = performance.now();
	
	let primes = [];

	for (let i = 0; i < 15000; i++) {
		let factors = [];

		for (let j = 1; j <= i; j++) {
			if (i % j == 0) {
				factors.push(j);
			}
		}

		if (factors.length == 2) {
			primes.push(i);
		}
	}

	console.dir(primes, {'maxArrayLength': null});
		
	console.log(`Time taken: ${(performance.now()-start)/1000}s`);
}

module.exports.primes = primes;
// for NodeJS